manifests/k/KRTirtho/Spotube/2.7.0/KRTirtho.Spotube.installer.yaml

```
@@ -0,0 +1,24 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: KRTirtho.Spotube
+PackageVersion: 2.7.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+ReleaseDate: 2023-03-07
```

```suggestion
ReleaseDate: 2023-03-07
AppsAndFeaturesEntries: 
  - DisplayName: Spotube version 2.7.0+17
    DisplayVersion: 2.7.0+17
    Publisher: 
    ProductCode: '80B901C8-D6FE-494E-8AF7-A2BD440E8644_is1'
```
Because `PackageVersion` doesn't match the version written to control panel, `AppsAndFeaturesEntries` are required

---

Added the suggestions in a commit